src/snowcli/cli/common/snow_cli_typer.py

```diff
@@ -0,0 +1,227 @@
+from __future__ import annotations
+
+from functools import wraps
+from typing import Any, Callable, Dict, Optional, Type, Union
+
+from click import ClickException
+from typer import Typer
+from typer.core import TyperCommand, TyperGroup, MarkupMode
+from typer.models import (
+    CommandFunctionType,
+    Default,
+)
+
+from snowcli.cli.common.snow_cli_global_context import snow_cli_global_context_holder
+
+
+def _are_tracebacks_enabled() -> bool:
+    return (
+        snow_cli_global_context_holder.get_global_context_copy().logging_and_exception_handling.enable_tracebacks
+    )
```

Comment:
Should this be a method of the holder class?

Reply:
I'm not sure about it.

I'd like to avoid adding more responsibilities to holder/manager. Let's have it only to provide and keep the context.

Reply:
With renaming to manager we get a good argument to move the method there. In future we may have more `is_something` methods and having them in one class may be better than multiple loose functions. WDYT? + @sfc-gh-astus @sfc-gh-jsikorski

Reply:
If we go this way, then we should also avoid calling `get_...` and `update_...` directly and wrap all reads and updates in helper methods.

Reply:
I agree with @sfc-gh-turbaszek, this function fits better in manager

Reply:
I don't understand why is that.

I created this function only to make the code more readable (to avoid long method execution chain).
I don't see any advantage of duplicating all fields stored in global context to manager's methods.

Reply:
1. Because this module has to know global context structure
2. `bool` is immutable so it can be return by manager and it won't be necessary to deepcopy whole context to get one value

Reply:
As we agreed to not use the solution from this PR, I made changes from this comment in new PR: https://github.com/Snowflake-Labs/snowcli/pull/192